app/helpers/deployments_helper.rb

```diff
@@ -1,7 +1,7 @@
 module DeploymentsHelper
   def show_deployment(deployment)
     display = deployment.display_attr(:integration_type)
-    display += " • #{deployment.build_artifact_channel["name"]}" if deployment.display_channel?
+    display += " • #{deployment.deployment_channel_name}" if deployment.display_channel?
```

Comment:
This should be routed through the `deployment_channel_name` helper right?

Reply:
I chose to use the model function since it existed, using a model function is more reliable than helper functions, imo.

Reply:
the helper function is used for deployment form where the model doesn't exist.